docs/algorithms.rst

```diff
@@ -172,7 +174,7 @@ are enlarged to a point where this error is negligible.
 Because this contribution is only accumulated when configurations are written to disk, the additional computational
 overhead is small.
 The largest allowable cutoff (slightly smaller than one-half the smallest box edge) is automatically selected for this
-purpose, but the user must specify a large enough box that the cutoff can be expanded to at least 16 Angstroms.
```

Comment:
We should probably create an issue to add the ability for YANK to do this automatically.